Makefile

```diff
@@ -9,38 +9,44 @@ else
        Q = @
 endif
 
-#VERSION = $(shell git describe --dirty --tags --always)
-#REPO = github.com/RHsyseng/console-cr-form
+# VERSION = $(shell git describe --dirty --tags --always)
+# REPO = github.com/RHsyseng/console-cr-form
 
-#export CGO_ENABLED:=0
+export GO111MODULE = on
```

Comment:
I don't think you need this with 12+ versions of Go. Want to double check?

Reply:
1.13+ is when it's no longer needed - https://golang.org/doc/go1.13#modules